lib/vagrant-libvirt/config.rb

```diff
@@ -129,6 +131,40 @@ def _get_device(disks)
 
       # NOTE: this will run twice for each time it's needed- keep it idempotent
       def storage(storage_type, options={})
+        if storage_type == :file
+          _handle_file_storage(options)
+        end
+      end
+
+      def _handle_file_storage(options={})
+        if options[:device] == :cdrom
+          _handle_cdrom_storage(options)
+        else
+          _handle_disk_storage(options)
+        end
+      end
+
+      def _handle_cdrom_storage(options={})
+        # <disk type="file" device="cdrom">
+        #   <source file="/home/user/virtio-win-0.1-100.iso"/>
+        #   <target dev="hdc"/>
+        #   <readonly/>
+        # </disk>
+
+        options = {
+          :dev => "hdc",
+          :bus => "ide",
+          :path => nil,
+        }.merge(options)
+
+        @cdrom = {
```

Comment:
eh, I hadn't tested it, and a single one fit my use case. Multiple cdroms should work. I'll test it out